R/bootstrap-model.R

```diff
@@ -0,0 +1,628 @@
+#' Create a boostrap model object from an existing model
+#'
+#' @param .mod a `bbr` model object
+#' @param .suffix a prefix for the boostrap model directory. Will be appended by
+#'  the boostrap run number for a given model.
+#' @inheritParams copy_model_from
+#' @param increment Logical (T/F). If `TRUE`, create a new bootstrap model run
+#'  from `.mod`. Useful for managing multiple bootstrap runs originating from
+#'  the same starting model. Will append `_run_[x]` to the provided `.suffix`.
+#'  Note that the`.overwrite` arg will have no impact if incrementing the
+#'  bootstrap run.
+#' @param remove_cov,remove_tables Logical (T/F). Optionally remove `$COVARIANCE`
+#' and `$TABLE` records respectively, allowing for notably faster run times.
+#'
+#' @seealso setup_bootstrap_run
+#' @return S3 object of class `bbi_nmboot_model`.
+#' @export
+new_bootstrap_run <- function(
+    .mod,
+    .suffix = glue("{get_model_id(.mod)}_boot"),
+    .inherit_tags = TRUE,
+    .overwrite = FALSE,
+    increment = FALSE,
```

Comment:
I think this increment is going to result in just a lot of runs getting triggered. Instead, could we have the user supply a sub-run number?

Reply:
Mentioned above, but I opted to just remove the `increment` argument. So now the only way to make a new run would be to change the `.suffix` arg (may be a different name later)